.storybook/design-system-docs-decorator.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { StoryContext } from '@storybook/react/*';
+import { ArrowIcon } from '../packages/components/src/components/icons';
+import { hiveThemeDecorator } from './hive-theme-decorator';
+
+export function designSystemDocsDecorator(Story: () => React.ReactNode, ctx: StoryContext) {
+  return hiveThemeDecorator(
+    () => (
+      <div className="h-screen overflow-auto bg-white p-8 text-green-1000 xl:p-16 dark:bg-green-1000 dark:text-white">
+        <header className="flex flex-row items-center text-5xl tracking-[-0.288px]">
+          <HiveIconMark className="mr-6" />
+          <p>Core Elements</p>
+          <ArrowIcon className="mx-4 shrink font-medium" />
+          <h1 className="font-medium">{ctx.title.split('/').pop()}</h1>
+          <p className="ml-auto text-sm leading-6 tracking-[-0.084px]">Hive</p>
+        </header>
+        <hr className="my-6 border-t border-t-beige-200" />
+        <section className="flex flex-col xl:px-6">
+          <Story />
+        </section>
+      </div>
+    ),
+    ctx,
+  );
+}
+
+function HiveIconMark(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
to `.svg`

Reply:
This is a storybook-only file. Importing the icon through `react-svg-loader` / `@svgr/webpack` won't break anything, but do we need a separate file for that? I'm doubtful if it's gonna be used elsewhere. And yeah, it is different to `icons/hive.svg`.

Reply:
it's to have consistency and avoid repeating declaring `props`, `{...props}` and it's types